jOOQ-examples/jOOQ-spring-boot-example/pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <groupId>org.jooq</groupId>
     <artifactId>jooq-spring-boot-example</artifactId>
-    <version>${org.jooq.version}</version>
+    <version>${jooq.version}</version>
```

Comment:
What if I fix a really really nasty bug in say jOOQ 3.17.348 but Spring Boot is still on 3.17.342 and when Spring Boot upgrades to 3.17.348, they inadvertedly also upgrade from Hibernate 4 to 6? :)

Dependencies.

OK, would you mind creating a separate issue for this?